core/src/main/scala/chisel3/IntModule.scala

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.experimental.{BaseModule, Param}
+import chisel3.internal.BaseIntModule
+import chisel3.internal.Builder.pushCommand
+import chisel3.internal.firrtl._
+import scala.annotation.nowarn
+
+package internal {
+
+  private[chisel3] abstract class BaseIntModule(intrinsicName : String) extends BaseModule {
+    val intrinsic = intrinsicName
+  }
+
+}
+
+package experimental {
```

Comment:
This structure of using a `package` here is a bit non-standard. I'd refactor this to be in a separate directory called `experimental` with a package line of `package chisel3.experimental`.

Reply:
This is ported directly from BlackBox.